com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointFloating.cs

```diff
@@ -23,27 +23,20 @@ public class UrdfJointFloating : UrdfJoint
         public static UrdfJoint Create(GameObject linkObject)
         {
             UrdfJointFloating urdfJoint = linkObject.AddComponent<UrdfJointFloating>();
-            #if UNITY_2020_1_OR_NEWER
-                urdfJoint.unityJoint = linkObject.AddComponent<ArticulationBody>();
-                //Doesnt have any equivalent Articulatiob Joint
-            #else
-            urdfJoint.UnityJoint = linkObject.AddComponent<ConfigurableJoint>();
-            #endif
+            urdfJoint.unityJoint = linkObject.AddComponent<ArticulationBody>();
+
             return urdfJoint;
         }
 
 #region Runtime
 
         public override float GetPosition()
         {
-            #if UNITY_2020_1_OR_NEWER
-                Debug.Log("'ArticulationBody' does not contain a definition for 'connectedAnchor' and no accessible extension method 'connectedAnchor'");
-                Vector3 distanceFromAnchor = ((ArticulationBody)unityJoint).transform.localPosition ;/*-
-                                         ((ArticulationBody)UnityJoint).connectedAnchor;*/
-            #else
-            Vector3 distanceFromAnchor = ((ConfigurableJoint)UnityJoint).transform.localPosition - 
-                                         ((ConfigurableJoint)UnityJoint).connectedAnchor;
-            #endif
+            // not true, weird - talk to Vidur
```

Comment:
@vidurvij-Unity ^^

Reply:
This was an oversight. But we need to make changes to correct this. Since this is a planar joint we need to return a pari of floats.

Reply:
Maybe we could collab on a proper way to fix this? I don't directly see what values to return as a pair...

Reply:
I think we should have a discussion over zoom or slack on how to move this forward really. 

Clearly, GetPosition() / GetVelocity() / GetEffort() interface is limiting and, in a way, confusing. Once we work with ArtculationBodies, any getter should return an instance of ArticulationReducedSpace actually instead of a float. This will be float for revolute, float for prismatic and up to three floats for spherical. I don't yet understand how we map urdf to articulation, but my intention would be to expose ABs directly here and not try to insert an abstracting interface, as that one is only an obstacle from my pov. 

Having that said, I would suggest to open a ticket to redesign this bit. The behaviour is not broken nor improved by this PR.